docker/run-tests-sp.sh

```diff
@@ -29,6 +29,13 @@ esac
 # shellcheck source=../scripts/fetch_envars.sh
 /test/scripts/fetch_envars.sh "${ENVIRONMENT}" | tee /test/.env
 
+if [ "${SAM_STACK_NAME:-}" == "authentication-api" ]; then
+  if grep ^API_RP_URL= /test/.env; then
+    API_RP_URL=$(grep ^API_RP_URL= /test/.env | cut -d= -f2)
+    echo "RP_URL=${API_RP_URL}" | tee --append /test/.env
+  fi
+fi
```

Comment:
I'd prefer if we replaced any existing RP_URL value, rather than add a second one to the file.

would something like this work?

```bash
  if grep -E '^API_RP_URL=' /test/.env) ; then
    api_rp_url=$(grep -E '^API_RP_URL=' /test/.env | cut -d= -f2-)
  
    if grep -qE '^RP_URL=' /test/.env; then
      sed -i "s|^RP_URL=.*|RP_URL=${api_rp_url}|" /test/.env
    else
      echo -e "\nRP_URL=${api_rp_url}" >> /test/.env
    fi
  fi
```

Reply:
Slightly changed from the suggestion
```bash
if [ "${SAM_STACK_NAME:-}" == "authentication-api" ]; then
  if grep -qE '^API_RP_URL=' /test/.env; then
    API_RP_URL=$(grep ^API_RP_URL= /test/.env | cut -d= -f2-)

    if grep -qE '^RP_URL=' /test/.env; then
      sed -i '/^RP_URL=.*/d' /test/.env
    fi
    echo -e "RP_URL=${API_RP_URL}" | tee --append /test/.env
  fi
fi
```
Updating with sed is possible, but it does it quietly. Could lead to a debugging nightmare. 
So instead, using sed to find and delete lines containing RP_URL. Then an echo...tee to append the file and print what's changed in the stdout.

Reply:
done deal!